fvm/src/call_manager/default.rs

```diff
@@ -388,12 +507,23 @@ where
         s.exec_trace.push(trace);
     }
 
-    fn create_account_actor<K>(&mut self, addr: &Address) -> Result<ActorID>
+    /// Helper method to create an uninitialized actor due to a send.
+    fn create_actor_from_send(&mut self, addr: &Address, act: ActorState) -> Result<ActorID> {
+        let _ = self.charge_gas(self.price_list().on_create_actor(true))?;
+        let addr_id = self.state_tree_mut().register_new_address(addr)?;
+        self.state_access_tracker.record_lookup_address(addr);
```

Comment:
Why do we not record this as an Actor Update?

Reply:
1. `on_create_actor` charge only charges for storage and address assignment.
2. `set_actor` below will charge to actually _create_ the actor (possibly charging to read it as well).